package/gluon-ssid-changer/luasrc/lib/gluon/upgrade/500-ssid-changer

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site'
+
+local uci = require('simple-uci').cursor()
+
+if site.ssid_changer ~= nil then
+	local site_enabled = site.ssid_changer.enabled() or '1'
+
+	uci:section('ssid-changer', 'settings', 'settings', {
+		enabled           = uci:get('ssid-changer', 'settings', 'enabled') or site_enabled,
+		switch_timeframe  = site.ssid_changer.switch_timeframe() or '30',
+		first             = site.ssid_changer.first() or '5',
+		prefix            = site.ssid_changer.prefix() or 'FF_Offline_',
+		suffix            = site.ssid_changer.suffix() or 'nodename',
+		tq_limit_enabled  = site.ssid_changer.tq_limit_enabled() or false,
+		tq_limit_max      = site.ssid_changer.tq_limit_max() or 45,
+		tq_limit_min      = site.ssid_changer.tq_limit_min() or 35,
```

Comment:
I would go for required site.conf entries instead of magic default values.

Reply:
i wrote the complete opposite, that it should work with defaults only :D